frontend/src/ProjectManager/ProjectManagerScriptOptions.svelte

```diff
@@ -1,32 +1,3 @@
-<style>
-  button {
-    margin-bottom: 1em;
-    padding-top: 0.5em;
-    padding-bottom: 0.5em;
-    padding-left: 1em;
-    padding-right: 1em;
-    background-color: var(--main-bg-color);
-    color: var(--main-fg-color);
-    border: 1px solid var(--main-fg-color);
-    border-radius: 0;
-    font-size: smaller;
-    overflow: hidden;
-    box-shadow: none;
-  }
-
-  button:hover,
-  button:focus {
-    outline: none;
-    box-shadow: inset 1px 1px 0 var(--main-fg-color),
-      inset -1px -1px 0 var(--main-fg-color);
-  }
-
-  button:active {
-    box-shadow: inset 2px 2px 0 var(--main-fg-color),
-      inset -2px -2px 0 var(--main-fg-color);
-  }
-</style>
-
```

Comment:
Should these styles be added back if the button is still included and not switched to a `<Button>` component below?

Reply:
other way around, fixed